docs/conf.py

```diff
@@ -4,18 +4,16 @@
 # full list see the documentation:
 # http://www.sphinx-doc.org/en/master/config
 
+import os
+
 from datetime import datetime
+from sphinx.application import Sphinx
```

Comment:
`docs/conf.py` is the [configuration file for Sphinx](https://www.sphinx-doc.org/en/master/usage/configuration.html).  If we want to change something about how the documentation is configured, this is usually the file that we need to modify.